package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "faunadb",
-  "displayName": "FaunaDB",
-  "description": "FaunaDB extension",
-  "icon": "media/faunadb-extension-logo.png",
+  "name": "fauna",
```

Comment:
We should leave this as `faunadb` because this is the namespace we publish to on NPM. Unfortunately, there is another project currently using `fauna`, so we may be stick with this.